builds/e2e/longhaul.yaml

```diff
@@ -346,4 +346,128 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
-         
\ No newline at end of file
+          
+################################################################################
+  - job: setup_iotuap_amd64_moby
+################################################################################
+    displayName: Windows IoT Core AMD64 Moby (Setup Only)
```

Comment:
Just want to confirm, the reason we need to split into 2 jobs: setup and test.  Because we can't install x64 version of VSTS agent on Win IoT Core x64; is it right?  why it fails?

Reply:
Yes, we need a separate _Windows IoT Core AMD64 Moby (Setup Only)_ stage to install/update the iotedge service on the runner VM (Windows IoT Core x64), which causes a reboot.
Then we have the other stage (_Windows IoT Core AMD64 Moby_) that runs the Longhaul test bypassing installation of iotedge service.

Reply:
Same situation with Stress test - we need two stages to handle the reboot of the runner VM.

Reply:
Question: does "Setup only" step restart the machine?

Reply:
Yes, installation/update of the iotedge service on Windows IoT Core triggers a reboot (just like with the E2E pipeline).

Reply:
please put a comment in setup template file to mention about auto-reboot of the device after installation.  say iotedge-win7 run setup only step and iotedge-iotuap7 restarted; then the next job picked up by iotedge-win7 immediately and run.  is it possible that iotedge-iotuap7 doesn't completely start up?  If that's the reason, we should be able to merge these 2 jobs into 1. right?

Reply:
I was not able to handle the runner's reboot in one step. The step just fails if the device reboots. Therefore I came up with this two-step approach:
* the first step "Setup only" installs/updates the service and reboots the runner
* the second step picks up the rebooted device and runs the Longhaul test

If the runner VM does not completely start, the second step (that runs the test) will fail being unable to connect to the runner.

Added a comment explaining the two stages and the reboot.

Reply:
"...the first step "Setup only" installs/updates the service and reboots the runner..."; would you please show me where the code did the reboot for runner device?

"...If the runner VM does not completely start, the second step (that runs the test) will fail being unable to connect to the runner...." does this happen sometimes? how about have a step to detect where runner device is restarted?